qa/rpc-tests/test_framework/blockchainhelper.py

```diff
@@ -304,9 +298,4 @@ def send_certificate(self, sc_name, quality, referenced_height = None):
             custom_fields,
             [RANDOM_BITVECTOR])
 
-        sc_info["last_certificate_epoch"] = epoch_number
-
-        if is_non_ceasing_sidechain:
-            sc_info["last_referenced_height"] = ref_height
-
```

Comment:
Is there a reason for no longer updating there values in the `sc_info` dictionary after submitting the certificate?

Reply:
My mistake, thanks for catching this. Restored in 3518be16ce52befbdfc21398187b4961582107f2